### DISCUSSION TITLE
Can't install Copilot in VSCode, extension not shown

### DISCUSSION THREAD

**Discussion by zrayx** [community/community#19671] [Copilot Conversations] [answered]
When I search for "copilot" in VSCode extensions, I get "no extensions.".

When I type Ctrl-P and enter "ext install GitHub.copilot", I get "No extensions found.".

I am subscribed to Copilot, an I'm on day 54 of my 60 day free trial.

Other extensions work flawlessly and can be downloaded, e.g. CodeLLDB, rust-analyzer etc.

$ code --version
1.68.1
30d9c6cd9483b2cc586687151bcbcd635f373630
x64

Arch Linux package code-1.68.1-1

**Comment by davecheney** [comment, marked as answer]
Linux distos often ship with, so called, source built versions like VScodium or OSSCode. Are you sure you’re using VScode?

**Reply by ImUrX** [reply]
But they are vscode, whats the difference do tell?

**Comment by pentr32** [comment]
Are you sure that you are using the stable VSCode? I have seen similar posts and the problem was VSCode insider version. Try to fresh re-install VSCode and try to search again. You can also install it from browser. 
https://marketplace.visualstudio.com/items?itemName=GitHub.copilot

**Comment by zrayx** [comment]
It appears I had OSSCode installed, so thanks @davecheney. I installed the Microsoft binary from Arch Linux AUR (https://aur.archlinux.org/visual-studio-code-bin.git) and this works.
I could install the vsix file, but it didn't work. But thanks @pentr32  as well.
